clang/lib/Driver/Driver.cpp

```diff
@@ -993,6 +1028,41 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
     return;
   }
 
+  //
```

Comment:
We don't use start a comment block with `//\n`. Just delete the part before `We need to generate a SYCL toolchain if the user specified -fsycl.`